projects/kit/components/rating/rating.style.less

```diff
@@ -0,0 +1,77 @@
+:host {
+    position: relative;
+    display: inline-flex;
+
+    &._readonly,
+    &._disabled {
+        pointer-events: none;
+    }
+
+    &._disabled {
+        opacity: 0.5;
+    }
+}
+
+.t-rating-wrapper,
+.t-rating-interactive {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    display: inline-flex;
+    overflow: hidden;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    white-space: nowrap;
+}
+
+.t-rating-interactive:hover ~ .t-rating-wrapper {
+    visibility: hidden;
+}
+
+.t-rating-interactive {
+    z-index: 2;
```

Comment:
Why do we need z-index? We can just put stuff in the template in right order.

Reply:
But I have problem with `.t-rating-interactive:hover ~ .t-rating-wrapper`

Reply:
fixed